common/src/main/java/com/microsoft/identity/common/nativeauth/internal/controllers/NativeAuthMsalController.kt

```diff
@@ -1520,6 +1520,11 @@ class NativeAuthMsalController : BaseNativeAuthController() {
         oAuth2Strategy: NativeAuthOAuth2Strategy,
         parameters: SignUpStartCommandParameters
     ): SignUpStartApiResult {
+        LogSession.logMethodCall(
```

Comment:
@Yuki-YuXin  given that the aim of this PR was to clean up logging, what's the reason for introducing logs here? Did you find that these are helpful breadcrumbs to have when e.g. debugging issues?

Reply:
Here are some comments and discussion before in a legacy PR. https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2374
One of the comments was raising an idea that remove the logs from OAuth2Strategy, RequestProvider and RequestHandler, but have some logging in the API layer.

Reply:
Gotcha, thanks for reminder me. All good.